packages/account/src/Helpers/utils.js

```diff
@@ -0,0 +1,12 @@
+import { localize } from '@deriv/translations';
+
+export const generatePlaceholderText = selected_doc => {
```

Comment:
Can you leave a comment explaining on what are the possibilities that you should expect as argument and explain which placeholder is this.

Reply:
I have added an explanation